### PR TITLE
Changes made based on Migules comments on previous PR

### PR DESCRIPTION
**Pull request by rmothilal**
Changes made based on Migules comments on previous PR
Added /events endpoint to the swagger and created endpoint.
Added code to work with gRPC server with the event-sdk
Updated version

### CODE REVIEW

File: src/interface/swagger.json [hunk 1]
```diff
@@ -179,6 +231,161 @@
...
+                },
+                "to": {
+                    "type": "string",
+                    "description": "The recipient of the message"
+                },
+                "id": {
+                    "type": "string",
+                    "description": "Unique identifier for message"
+                },
+                "content": {
```
Comment (ramirogm) [inline src/interface/swagger.json]: I think property `pp` is missing here

Comment (rmothilal) [reply src/interface/swagger.json]: i didn't see it pp in the example https://github.com/mdebarros/documentation/blob/feature/%23767DesignEvent,AuditingandErrorFrameworkProposal/mojaloop-technical-overview/event-framework/README.md#31-json-example

Comment (ramirogm) [reply src/interface/swagger.json]: Is an optional property, see https://github.com/mdebarros/documentation/blob/feature/%23767DesignEvent%2CAuditingandErrorFrameworkProposal/mojaloop-technical-overview/event-framework/README.md#321-object-definition-eventmessage

Comment (rmothilal) [reply src/interface/swagger.json]: thanks

File: package.json [hunk 2]
```diff
@@ -26,10 +26,10 @@
         "temp-directory": "./.nyc_output",
         "check-coverage": true,
         "per-file": true,
-        "lines": 90,
-        "statements": 90,
-        "functions": 90,
-        "branches": 90,
+        "lines": 80,
```
Comment (mdebarros) [inline package.json]: why 80? The standard is 90.

Comment (mdebarros) [inline package.json]: Should this not be in a separate file?

File: src/index.js [hunk 3]
```diff
@@ -28,25 +28,23 @@ const Server = require('./server')
...
 
 Program
   .version(PJson.version)
   .description('CLI to manage Servers')
 
-Program.command('api') // sub-command name, coffeeType = type, required
-  .alias('a') // alternative sub-command is `o`
+Program.command('api')
+  .alias('a')
   .description('Start the Event Sidecar. Use options to specify server type of none to run both') // command description
```
Comment (mdebarros) [inline src/index.js]: I see no options here, or am I blind?

Comment (rmothilal) [reply src/index.js]: fixed

**Pull request merged**